api-mock/mock-server/utils/handleSession.ts

```diff
@@ -0,0 +1,14 @@
+import Session from "supertokens-node/recipe/session";
+
+export async function handleSession(
+    session: { [key: string]: any } | undefined
+): Promise<Session.SessionContainer | undefined> {
+    if (session !== undefined) {
+        // TODO: review this workaround
```

Comment:
The only issue I see is if a session expires this will throw and not return the session object w/ expired tokens. I'm not sure if we have any tests that rely on having a session object expire.

Reply:
ok cool, for these test batch is working fine